<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub1553_gkmaygsl</class>
 <widget class="QWidget" name="sub1553_gkmaygsl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1457</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox1</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>L频段天线子阵TR开关选择0x60</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="2" column="0">
            <widget class="QCheckBox" name="L_Submatrix3">
             <property name="text">
              <string>子阵3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="L_Submatrix4">
             <property name="text">
              <string>子阵4</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="L_Submatrix2">
             <property name="text">
              <string>子阵2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="L_Submatrix1">
             <property name="text">
              <string>子阵1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>子阵开关</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="on_PB_switch_clicked">
           <property name="text">
            <string>L频段天线子阵TR开关选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>内存下卸 0x61</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>启停标识</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="CB__0x61_1">
             <item>
              <property name="text">
               <string>停止</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>开始</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>起始地址</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_0x61_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>00000000</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>长度</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_0x61_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_SAR_end">
           <property name="text">
            <string>内存下卸</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>清空雷达指令包0x63</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_radar_clear">
             <item>
              <property name="text">
               <string>保留</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_radar_clear">
           <property name="text">
            <string>清空雷达指令包</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>删除单个雷达指令包0x64</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_single_delete">
             <item>
              <property name="text">
               <string>保留</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>SAR工作参数包号</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="LE_single_delete">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_single_delete">
           <property name="text">
            <string>删除单个雷达指令包</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>重要数据采集 0x65</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QFormLayout" name="formLayout_7">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_radar_modify">
             <item>
              <property name="text">
               <string>保留</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_radar_modify">
           <property name="text">
            <string>重要数据采集</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>GroupBox2</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>重要数据恢复 0x66</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QFormLayout" name="formLayout_8">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_radar_backup">
             <item>
              <property name="text">
               <string>保留</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>重要数据恢复</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="title">
         <string>清空所有上注的补丁包0x67</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QFormLayout" name="formLayout_9">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_clearall">
             <item>
              <property name="text">
               <string>保留</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_clearall">
           <property name="text">
            <string>清空所有上注的补丁包</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_12">
        <property name="title">
         <string>L频段阵面与通道对应关系0x68</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QFormLayout" name="formLayout_10">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_L_link">
             <item>
              <property name="text">
               <string>阵面1-4对应通道1234</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>阵面1-4对应通道1256</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>阵面1-4对应通道3456</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_L_link">
           <property name="text">
            <string>L频段阵面与通道对应关系</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_13">
        <property name="title">
         <string>导航闭锁开关0x69</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>导航闭锁</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_navi_locks">
             <item>
              <property name="text">
               <string>关</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>开</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_navi_lock">
           <property name="text">
            <string>导航闭锁开关</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_17">
        <property name="title">
         <string>L频段数字综合单元CPU软件补丁注入0x6A</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QPushButton" name="pushButton_6A_read">
           <property name="text">
            <string>读取数据文件</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_33">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_6A_path">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6A_in">
           <property name="text">
            <string>CPU软件补丁注入</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_18">
        <property name="title">
         <string>L频段数字综合单元CPU波控码0x6B</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <widget class="QPushButton" name="pushButton_6B_read">
           <property name="text">
            <string>读取数据文件</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_34">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_6B_path">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6B_order">
           <property name="text">
            <string>CPU波控码</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_16">
     <property name="title">
      <string>GroupBox3</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_14">
        <property name="title">
         <string>SSD盘还原 0x6C</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <layout class="QFormLayout" name="formLayout_15">
           <item row="0" column="0">
            <widget class="QLabel" name="label_45">
             <property name="text">
              <string>第1个区大小(512MB)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_ssdhy_1">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>第2个区大小(512MB)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_ssdhy_2">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_ssd_hy">
           <property name="text">
            <string>SSD盘还原</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_15">
        <property name="title">
         <string>温度关断保护使能0x71</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <layout class="QFormLayout" name="formLayout_13">
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="CB_T_protect">
             <item>
              <property name="text">
               <string>关</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>开</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_T_protect">
           <property name="text">
            <string>温度关断保护使能</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_19">
        <property name="title">
         <string>设备主备启动设置0x72</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>设备选择</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_0x72_1bit0_1">
             <property name="text">
              <string>可重构射频前端</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_0x72_1bit2_3">
             <property name="text">
              <string>L频段数字主控机</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_0x72_1bit4_5">
             <property name="text">
              <string>L频段定时波控机</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_0x72_1bit6_7">
             <property name="text">
              <string>SXKu波束控制机</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>可重构射频前端主备</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_0x72_2">
             <item>
              <property name="text">
               <string>主</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>备</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_40">
             <property name="text">
              <string>L频段数字主控机主备</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboBox_0x72_3">
             <item>
              <property name="text">
               <string>主</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>备</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string>L频段定时波控机主备</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="comboBox_0x72_4">
             <item>
              <property name="text">
               <string>主</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>备</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string>SXKu波束控制机主备</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="comboBox_0x72_5">
             <item>
              <property name="text">
               <string>主</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>备</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_41">
             <property name="text">
              <string>设备选择2</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="label_42">
             <property name="text">
              <string>：</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="checkBox_0x72_6bit2_3">
             <property name="text">
              <string>可重构射频前端高相噪</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCheckBox" name="checkBox_0x72_6bit0_1">
             <property name="text">
              <string>可重构射频前端二本振</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QComboBox" name="comboBox_0x72_7">
             <item>
              <property name="text">
               <string>主</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>备</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_43">
             <property name="text">
              <string>可重构射频前端二本振主备</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_44">
             <property name="text">
              <string>可重构射频前端高相噪主备</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QComboBox" name="comboBox_0x72_8">
             <item>
              <property name="text">
               <string>主</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>备</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_0x72">
           <property name="text">
            <string>设备主备启动设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
